### PR TITLE
update issue template

### PR DESCRIPTION
**Pull request by MetalBlueberry**
After #4906 discussion, here is the merge request.

This aims to standardize the issue names and bodies.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -3,9 +3,36 @@ name: "\U0001F41B Bug Report"
...
 
-## Bug Report
+# Bug Report
 
-### Please provide information about your setup
+## Issue name
+
+Issue names must follow the pattern `command: description` where the command is the dvc command that you are trying to run. The description should describe the consequence of the bug. Example: `repro: doesn't detect input changes`
+
+Delete this section before publish
```
Comment (pmrowla) [inline .github/ISSUE_TEMPLATE/bug_report.md:12]: This kind of thing can be done with comments, so the user will see the message when they open the bug report, but don't have to explicitly remove all of the instructions every time they file a report.

the template would just look something like
```
# Bug Report

<!--
Issue names should follow the pattern ...
Example: `repro: doesn't detect input changes`
-->

...
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -3,9 +3,36 @@ name: "\U0001F41B Bug Report"
...
+
+A clear and concise description of what the bug is.
+
+### Reproduce
+
+Step list of how to reproduce the bug
+
+1. dvc init
+2. Copy dataset.zip to the directory
+3. dvc add dataset.zip
```
Comment (pared) [inline .github/ISSUE_TEMPLATE/bug_report.md:24]: maybe lets add "for example"

**Review by pared — APPROVED** [review]
Looks good to me!
I just wonder whether users will be reading the template, but I guess we won't know until we try.

**Review by karajan1001 — APPROVED** [review]
SGTM.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (MetalBlueberry) [reply .github/ISSUE_TEMPLATE/bug_report.md]: This is a good idea! I basically added this to information blocks so they do not appear by default.

**Pull request merged**